tox.ini

```diff
@@ -41,18 +38,32 @@ commands =
     #   Python 3.5 environments that bundle an older version of virtualenv.
     python -bb \
         -Werror -Wignore:::_pytest.assertion.rewrite -Wignore:::_pytest.tmpdir -Wignore:::site \
-        -m pytest --cov=fact --cov-report=html
+        -m pytest --cov="{envsitepackagesdir}/fact"
+
+[testenv:coverage]
+skip_install = true
+depends = py27,py35,py36,py37
+setenv =
```

Comment:
Is this `setenv` line necessary?

Reply:
This cancels out the setting of `COVERAGE_FILE` in `[testenv]`. It is necessary on `[testenv:coverage]` so that it reverts to the default of `COVERAGE_FILE=.coverage`. It is not strictly necessary on `pep8` and `docs`, but I put it there so unnecessary variables are not set in those environments.

Reply:
A comment about this inline would be useful.